docs/pages/tutorials/minimal/extend_schema.mdx

```diff
@@ -0,0 +1,329 @@
+import { CollapseCode } from "../../../components/CollapseCode";
+
+# Extend the schema
+
+In this tutorial you extend the schema to add a table of historical counter values and the time in which the counter reached those values.
+For the sake of simplicity, we will implement this in the `increment` function rather than use a [storage hook](/store/advanced-features#storage-hooks).
+
+## Setup
+
+Right now you need the `main` version of `mud`, not `next`.
+So if you followed the directions, you need to run this command in the root directory.
+
+```sh copy
+pnpm mud set-version --tag main && pnpm install
+```
+
+## Modify the MUD configuration file
+
+1. In an editor, open `packages/contracts/mud.config.ts` and add a table definition for `History`.
+
+   <CollapseCode>
+
+   ```ts filename="mud.config.ts" showLineNumbers copy {9-17}
+   import { mudConfig } from "@latticexyz/world/register";
+
+   export default mudConfig({
+     tables: {
+       Counter: {
+         keySchema: {},
+         schema: "uint32",
+       },
+       History: {
+         keySchema: {
+           counterValue: "uint32",
+         },
+         schema: {
+           blockNumber: "uint256",
+           time: "uint256",
+         },
+       },
+     },
+   });
+   ```
+
+   </CollapseCode>
+
+<details>
+<summary>Explanation</summary>
+
+A MUD table has two schemas:
+
+- `keySchema`, the key used to find entries
+- `schema`, the value in the entry (soon to be renamed to `valueSchema`)
+
+Each schema is represented as a structure with field names as keys, and the appropriate [Solidity data types](https://docs.soliditylang.org/en/latest/types.html) as their values.
+
+In this case, the counter value is represented as a 32 bit unsigned integer, because that is what `Counter` uses.
+Block numbers and timestamps can be values up to `uint256`, so we'll use this type for these fields.
+
+</details>
```

Comment:
should we indent this to the same level as the `1.`?

Reply:
Yes, we *should*. However, when I do it the prettier script breaks it. :-( 

When I try to  indent and then commit, `prettier` "fixes" it to this:

```html

    <details>
 <summary>Explanation</summary>

   A MUD table has two schemas:

   - `keySchema`, the key used to find entries
   - `schema`, the value in the entry (soon to be renamed to `valueSchema`)

   Each schema is represented as a structure with field names as keys, and the appropriate [Solidity data types](https://docs.soliditylang.org/en/latest/types.html) as their values.

   In this case, the counter value is represented as a 32 bit unsigned integer, because that is what `Counter` uses.
   Block numbers and timestamps can be values up to `uint256`, so we'll use this type for these fields.

    </details>

```


And then I get this error:

```
./pages/tutorials/minimal/add-table.mdx
Expected a closing tag for `<details>` (42:5-42:14) before the end of `listItem`
```

I spent enough time on it that I decided it's not worth hassling with.

Reply:
I _think_ the trick to get it to work is add a blank line between the opening tag and its contents, like we did for the `<CollapsedCode>`.